pyupgrade.py

```diff
@@ -1197,6 +1228,20 @@ def _replace_attr(i, mapping, node):
                 func_args, end = _parse_call_args(tokens, i + 1)
                 template = SIX_CALLS[node.func.id]
                 _replace_call(tokens, i, end, func_args, template)
+        elif token.offset in visitor.literal_attrs:
+            node = visitor.literal_attrs[token.offset]
+            template = 'b{args[0]}'
+            if tokens[i + 1].src == '(' and _is_ascii(tokens[i + 2].src):
+                func_args, end = _parse_call_args(tokens, i + 1)
+                _replace_call(tokens, i, end, func_args, template)
+            elif (
+                    tokens[i + 1].src == '.' and
+                    tokens[i + 2].src == node.func.attr and
+                    tokens[i + 3].src == '(' and
+                    _is_ascii(tokens[i + 4].src)
+            ):
```

Comment:
ugly, but I think this current breaks:

```
six.b(    "eurosymbol")
```

Reply:
I see. Do the other six fixers handle this?

Reply:
yes, this case differs because you have to inspect the arguments